manim/mobject/coordinate_systems.py

```diff
@@ -327,21 +338,39 @@ def get_lines(self):
         lines2 = VGroup(*x_lines2, *y_lines2)
         return lines1, lines2
 
-    def get_lines_parallel_to_axis(self, axis1, axis2, freq, ratio):
-        line = Line(axis1.get_start(), axis1.get_end())
+    def get_lines_parallel_to_axis(self, axis_parallel_to, axis_perpendicular_to, freq, ratio):
+        """Generated a set of lines parallel to an axis.
+
+        Parameters
+        ----------
+        axis_parallel_to : :class:`~.Line`
+            The axis with which the lines will be parallel.
+
+        axis_perpendicular_to : :class:`~.Line`
+            The axis with which the lines will be perpendicular. 
+
```

Comment:
Parameters `freq` and `ratio` aren't documented, so might want to include those instead of just making a partial documentation

Reply:
Done, thank you Captain Docs !